### PR TITLE
[fix bug 1115378] Add Mozilla Pontoon profile to external account

### PR DESCRIPTION
**Pull request by swarnava**

### CODE REVIEW

**Comment by akatsoulas** [comment]
@johngian r?

File: mozillians/users/models.py [hunk 1]
```diff
@@ -835,6 +836,9 @@ class ExternalAccount(models.Model):
         TYPE_MOLOCOMOTION: {'name': 'Mozilla Locomotion',
                             'url': 'http://mozilla.locamotion.org/accounts/{identifier}/',
                             'validator': validate_username_not_url},
+        TYPE_MOPONTOON: {'name': 'Mozilla Pontoon',
+                            'url': 'https://pontoon.mozilla.org/users/{identifier}/',
```
Comment (johngian) [inline mozillians/users/models.py]: Please fix identation here.

File: mozillians/users/models.py [hunk 2]
```diff
@@ -835,6 +836,9 @@ class ExternalAccount(models.Model):
         TYPE_MOLOCOMOTION: {'name': 'Mozilla Locomotion',
                             'url': 'http://mozilla.locamotion.org/accounts/{identifier}/',
                             'validator': validate_username_not_url},
```
Comment (johngian) [inline mozillians/users/models.py]: It looks like Mozilla Pontoon uses as an identifier for the user url the profile's email. We can change the validator here to ``validate_email``.

File: mozillians/users/models.py (hunk 1 above)
Comment (swarnava) [inline mozillians/users/models.py]: Can you tell me more about this? looks like they are using pontoon.mozilla.org/contributors/{identifier}/ now!

**Comment by johngian** [comment]
It might have not been clear in my previous comment, but the issue here is that the URL that we render to redirect users to pontoon.mozilla.org get's escaped and for example instead of:

 ``https://pontoon.mozilla.org/contributor/swarnavasengupta@example.com/``

we redirect to 

``https://pontoon.mozilla.org/contributor/swarnavasengupta%40example.com/``

**Comment by swarnava** [comment]
but https://pontoon.mozilla.org/contributor/swarnavasengupta@example.com/ is working fine for me, can you tell me the fix instead? :)

File: mozillians/users/models.py [hunk 3]
```diff
@@ -835,6 +836,9 @@ class ExternalAccount(models.Model):
...
+        TYPE_MOPONTOON: {'name': 'Mozilla Pontoon',
+                            'url': 'https://pontoon.mozilla.org/contributor/{identifier}/',
```
Comment (johngian) [inline mozillians/users/models.py]: Please indent here.

Comment (johngian) [inline mozillians/users/models.py]: Same here.

**Comment by swarnava** [comment]
@johngian  Looks good?

**Comment by johngian** [comment]
lgtm r+ :rocket:

**Comment by johngian** [comment]
Merging it here #1155 after some minor fixes.

**Pull request closed without merging**